README.md

```diff
@@ -4,20 +4,44 @@
 
 **Ru**by **fo**rmatter
 
-## Why
 
-Rufo's primary use case is as a text-editor plugin, to autoformat files on save or
-on demand. For this reason it needs to be fast. If the formatter is slow, saving
-a file becomes painful.
+Rufo is as an _opinionated_ ruby formatter, intended to be used via the command line as a text-editor plugin, to autoformat files on save or on demand.
 
-Right now, Rufo can format it's [formatter](https://github.com/ruby-formatter/rufo/blob/master/lib/rufo/formatter.rb),
-a 3000+ lines file, in about 290ms. It can format a ~500 lines file in 180ms. Since most files
-have less than 500 lines, the time is acceptable.
+Unlike the best known Ruby formatter [RuboCop](https://github.com/bbatsov/rubocop), Rufo offers little in the way of configuration. Like other language formatters such as [gofmt](https://golang.org/cmd/gofmt/), [prettier](https://github.com/prettier/prettier), and [autopep8](https://github.com/hhatto/autopep8), we strive to find a "one true format" for Ruby code, and make sure your code adheres to it, with zero config where possible.
+
+RuboCop does much more than just format code though, so feel free to run them both!
+
+Rufo supports all Ruby versions >= 2.2.4, but works most reliably with >= 2.3.**5** / >= 2.4.**2**, due to a bug in Ruby's Ripper parser.
+
+## Installation
+
+Add this line to your application's Gemfile:
+
+```ruby
+gem 'rufo'
+```
+
+And then execute:
+
+    $ bundle
+
+Or install it system wide with:
+
+    $ gem install rufo
+
+## Editor support
+
+Once the gem is installed, enable format on save integration in your editor of choice with the following libraries:
+
+- Atom: [rufo-atom](https://github.com/bmulvihill/rufo-atom) :construction:
+- Emacs [emacs-rufo](https://github.com/aleandros/emacs-rufo) :construction: or [rufo.el](https://github.com/danielma/rufo.el)
+- Sublime Text: [sublime-rufo](https://github.com/ruby-formatter/sublime-rufo)
+- Vim: [rufo-vim](https://github.com/splattael/rufo-vim)
+- Visual Studio Code: [vscode-rufo](https://marketplace.visualstudio.com/items?itemName=mbessey.vscode-rufo) or [rufo-vscode](https://marketplace.visualstudio.com/items?itemName=siliconsenthil.rufo-vscode)
+
+If you're interested in developing your own plugin check out the [development docs](docs/developing-rufo.md). Did you already write a plugin? That's great! Let us know about it and
+we will list it here.
 
-There's at least one other good Ruby formatter I know: [RuboCop](https://github.com/bbatsov/rubocop).
-However, it takes between 2 and 5 seconds to format a 3000+ lines file, and about 1 second to format
-a 500 lines file. A second is too much delay for a plugin editor. Additionally, RuboCop is much more
-than just a code formatter. Rufo is and will always be a code formatter.
 
 ## Unobtrusive by default
```

Comment:
Does this title still make sense with some of the recent changes to make it more opinionated?

Reply:
Yes I was a little conflicted, I did read through the contents of this section and check to ensure rufo still does all these things, and it does! So it does seem appropriate still. Despite enforcing quite a lot more than before, it is still very permissive in this area.